mslib/msui/mscolab.py

```diff
@@ -231,6 +234,20 @@ def connect_handler(self):
                 self.loginEmailLe.setEnabled(True)
                 self.loginPasswordLe.setEnabled(True)
 
+                try:
+                    idp_enabled = json.loads(r.text)["USE_SAML2"]
+                except (json.decoder.JSONDecodeError, KeyError):
+                    idp_enabled = False
+
+                if idp_enabled:
+                    # Hide user creatiion seccion if IDP login enabled
+                    self.addUserBtn.setHidden(True)
+                    self.clickNewUserLabel.setHidden(True)
```

Comment:
also in the workflow of profile/usercreation by the IDP is not completed. The new profile is not activated, it would need a validation (recover password attempt currently) first. There some changes we have to think about after we merged this in.

Reply:
We have to verify too that a user with a "None" Password because of IDP can't fetch one and update that 'None' over password recovery.  Some of these parts need a coverage by tests.